pkg/proxy/apis/config/validation/validation.go

```diff
@@ -42,7 +42,7 @@ func Validate(config *kubeproxyconfig.KubeProxyConfiguration) field.ErrorList {
 
 	effectiveFeatures := utilfeature.DefaultFeatureGate.DeepCopy()
 	if err := effectiveFeatures.SetFromMap(config.FeatureGates); err != nil {
-		allErrs = append(allErrs, field.Invalid(newPath.Child("featureGates"), config.FeatureGates, err.Error()))
+		allErrs = append(allErrs, field.Invalid(newPath.Child("FeatureGates"), config.FeatureGates, err.Error()))
```

Comment:
Does the configuration object not follow the camel casing used in the rest of api defs? @aojea ?

Reply:
this appears to be the only match for `newPath.Child\("[a-z]` in the entire tree, so it looks like using Go names rather than JSON names is correct here